lib/arc_ecto/schema.ex

```diff
@@ -53,4 +22,37 @@ defmodule Arc.Ecto.Schema do
 
     end) || params
   end
+
+  # If given a changeset, apply the changes to obtain the underlying data
+  # It would be nice to know under what context this will not be a changeset.
+  defp normalize_scope changeset_or_data do
+    case changeset_or_data do
+      %Ecto.Changeset{} -> Ecto.Changeset.apply_changes(changeset_or_data)
+      %{__meta__: _} -> changeset_or_data #TODO: wont this cause normalize_params to fail?
```

Comment:
All Ecto structs have a `__meta__` attribute, so `%{__meta__: _}` matches against all ecto structs.

There's probably a better way to do this now, but this was all I could find to reasonably match an Ecto schema struct back then.